src/core/hle/service/http_c.cpp

```diff
@@ -48,6 +50,74 @@ const ResultCode ERROR_WRONG_CERT_HANDLE = // 0xD8A0A0C9
 const ResultCode ERROR_CERT_ALREADY_SET = // 0xD8A0A03D
     ResultCode(61, ErrorModule::HTTP, ErrorSummary::InvalidState, ErrorLevel::Permanent);
 
+void Context::MakeRequest() {
+    assert(state == RequestState::NotStarted);
```

Comment:
```suggestion
    ASSERT(state == RequestState::NotStarted);
```
`common/assert.h` is required